funnel/templates/profile.html.jinja2

```diff
@@ -155,56 +155,55 @@
+            <div class="grid">
+              <div class="grid__col-md-7 grid__col-lg-8 mui--hidden-xs mui--hidden-sm mui--hidden-md">
+                {% if has_membership_project.bg_image.url %}
+                  <img class="img-fit img-rounded-border js-lazyload-img" data-src="{{ has_membership_project.bg_image.resize(img_size.profile_banner) }}" alt="{% trans %}Membership{% endtrans %}"/>
+                  <noscript>
+                    <img class="img-fit img-rounded-border" src="{{ has_membership_project.bg_image.resize(img_size.profile_banner) }}" alt="{% trans %}Membership{% endtrans %}"/>
+                  </noscript>
+                {% else %}
+                  <img class="img-fit img-rounded-border" src="{{ url_for('static', filename='img/default-banner.png') }}" alt="{% trans %}Membership{% endtrans %}"/>
+                {% endif %}
               </div>
-              <div class="grid__col-md-1 grid__col-lg-1 mui--hidden-xs mui--hidden-sm mui--hidden-md">
-              </div>
-              <div class="grid__col-12 grid__col-md-5 grid__col-lg-4" data-cy="spotlight-project">
-                <div class="card card--upcoming">
-                  <div class="card__image-wrapper {% if not has_membership_project.bg_image.url %}card__image-wrapper--default{% endif %}">
-                    <div>
+              <div class="grid__col-12 grid__col-md-5 grid__col-lg-4">
+                <div class="card card--shaped  mui--hidden-lg mui--hidden-xl">
+                  <div class="card__image-wrapper">
                     {% if has_membership_project.bg_image.url %}
                       <img class="card__image js-lazyload-img" data-src="{{ has_membership_project.bg_image.resize(img_size.card_banner) }}" alt="{% trans %}Membership{% endtrans %}"/>
                       <noscript>
-                        <img class="card__image" src="{{ has_membership_project.bg_image.resize(img_size.card_banner) }}" alt="{{ has_membership_project.title }}"/>
+                        <img class="card__image" src="{{ has_membership_project.bg_image.resize(img_size.card_banner) }}" alt="{% trans %}Membership{% endtrans %}"/>
                       </noscript>
                     {% else %}
-                      <img class="card__image" src="{{ url_for('static', filename='img/default-banner.png') }}" alt="{{ has_membership_project.title }}"/>
-                      <p class="card__image__tagline mui--text-body2">{% trans %}Membership{% endtrans %}</p>
+                      <img class="card__image" src="{{ url_for('static', filename='img/default-banner.png') }}" alt="{% trans %}Membership{% endtrans %}"/>
                     {% endif %}
-                    </div>
                   </div>
                   <div class="card__body">
-                    <div class="markdown mui--hidden-lg mui--hidden-xl">{{ has_membership_project.description.html }}</div>
+                    <div class="markdown">{{ has_membership_project.description.html }}</div>
```

Comment:
Avoid `.html` unless you're dealing with JSON content. The field will render as HTML.